LDSwiftEventSource.xcodeproj/project.pbxproj

```diff
@@ -239,6 +239,7 @@
 /* Begin PBXShellScriptBuildPhase section */
 		B46C1C6B24CF348B00283630 /* Linter Script */ = {
 			isa = PBXShellScriptBuildPhase;
+			alwaysOutOfDate = 1;
```

Comment:
This is a linting fix. We run a script as part of the build process. By default, it expects we will produce some artifacts it can use to determine if it should run again (like make). Setting this to 1 indicates we don't produce those artifacts and that we expect it to run each time.